src/components/Nav/Footer/Footer.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import styled from "styled-components";
+import { FaFacebook } from "react-icons/fa";
+import { FaInstagram } from "react-icons/fa";
```

Comment:
```suggestion
import { FaFacebook ,  FaInstagram} from "react-icons/fa";
```
같은 곳에서 import하는거면, `,`로 구분해서 선언해주는 게 깔끔합니다!

Reply:
수정 완료했습니다!